### PR TITLE
feat: add filter for multi-routing sort preference

### PR DESCRIPTION
**Pull request by mikasackermn**
# Summary

User could change sort preference in routes page.
(Added sort filter for multi routing)

Fixes # (issue)

# Checklist:

- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] Implemented a user interface (UI) change, referencing our Figma design to ensure pixel-perfect precision.

### CODE REVIEW

**Review by yeager-eren — CHANGES_REQUESTED** [review]
Thanks. I reviewed your PR, in addition to comments I put, these two items has been missed:

1. Hover on items hasn't been implemented
2. On dark mode, hovering on select has a white color.

File: widget/ui/src/components/Select/Select.types.ts [hunk 1]
```diff
@@ -5,6 +5,6 @@ type Item = {
 export type PropTypes = {
   options: Item[];
   handleItemClick?: (item: Item) => void;
-  value: Item;
+  value?: Item;
```
[the remark this answers is not in the document]
Comment (mikasackermn) [reply widget/ui/src/components/Select/Select.types.ts:8]: fixed

File: widget/ui/src/components/Select/select.styles.ts [hunk 2]
```diff
@@ -9,92 +9,61 @@ export const SelectTrigger = styled(Select.Trigger, {
...
   '& svg': {
     transition: `all ${EXPANDABLE_QUOTE_TRANSITION_DURATION}ms ease`,
   },
-
+  '& ._typography': {
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    width: '52px',
+    '@xs': {
```
[the remark this answers is not in the document]
Comment (mikasackermn) [reply widget/ui/src/components/Select/select.styles.ts:30]: This was before, when we were going to use select on the first page, in the size of 375 px was messed up. We don't need it now.

**Review by yeager-eren — APPROVED** [review]
Thanks. lgtm.

**Pull request merged**